gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/JobEvent.java

```diff
@@ -31,6 +31,7 @@
   public static final String LOCK_IN_USE = "LockInUse";
   public static final String WORK_UNITS_MISSING = "WorkUnitsMissing";
   public static final String WORK_UNITS_EMPTY = "WorkUnitsEmpty";
+  public static final String WORK_UNITS_CREATED = "WorkUnitsCreated";
```

Comment:
seems like these event name constants are scattered between `JobEvents` class and `EventNames` enum.  should we combine them?

Reply:
EventName enum seems to be defined specifically for Job Timers. I am not sure if this is by design or it just happened to be that way..